assets/src/stories-editor/components/higher-order/with-right-click-handler.js

```diff
@@ -57,8 +57,22 @@ const applyWithSelect = withSelect( ( select, props ) => {
 		}
 		const relativePositionX = event.clientX - wrapperDimensions.left;
 		const relativePositionY = event.clientY - wrapperDimensions.top - toolBarHeight;
+		const clientID = getCurrentPage();
+
+		let insidePercentageY = 0;
+		let insidePercentageX = 0;
+
+		const block = getBlockDOMNode( clientID );
```

Comment:
I'd rename this variable to `page`. Otherwise it is not clear which block this refers to.

Reply:
Fixed in https://github.com/ampproject/amp-wp/pull/3220/commits/e308d19cf26ef854254099abaa8ce29d6281f91d